### DISCUSSION TITLE
关于文本分类，你遇到过什么难题、有哪些需求、实战经验？

### DISCUSSION THREAD

**Discussion by chenxiaozeng** [PaddlePaddle/PaddleNLP#3310] [Ideas]
近日，PaddleNLP开源 [文本分类全流程应用方案](https://github.com/PaddlePaddle/PaddleNLP/blob/develop/applications/text_classification) ，覆盖多分类、多标签、层次分类各类场景，支持小样本学习 和 TrustAI 可信计算模型训练与调优，新增基于检索的思路完成分类任务的创新方案，无需标注，新增类别标签无需重训模型。

关于文本分类，你还有哪些需求、遇到过什么难题、有哪些实战经验，欢迎交流。

相关文章：https://mp.weixin.qq.com/s/tas7yM8vapxwtlJt-MRZdg

**Comment by wj-Mcat** [comment]
欢迎各位提出在文本分类场景下面临的各种问题，在此讨论当中将会有来自诸多专业团队的人员的讨论和回复，也欢迎各位参与此讨论当中来。

大家可按照以下格式来发起讨论topic：

```markdown
# <topic-name>

## 背景介绍

> 描述topic背景，以便让其它开发者能够迅速了解其问题本源，比如：少样本对话场景下的文本意图识别

## 核心问题

> 无论是需求还是难题，都可在此处作详细描述，比如：少样本的效果很差，并给出自己的实现方案，如果有github repo 就附上链接。

## 预期输出

> 描述你对于此topic的目标预期，比如：使用五个样本即可实现一定程度上的精准分类。

```

**Comment by datalee** [comment]
1、没有标注，或者标注很少，冷启动如何快速迭代至可用状态
2、对可以商用的要求又很高，没见过的数据至少要80%准确率以上
3、模型随机性不可控，每次都可能在领导关心的case上出问题；第一次出问题可以理解，我已经告诉你这个错了，第二次还错就不能原谅
4、模型小样本下抗干扰差，很容易受到非核心符号影响：比如句号等
5、有时候模型很傻，不具备知识和推理
